module.json

```diff
@@ -12,6 +12,11 @@
             "lang": "en",
             "name": "English",
             "path": "lang/en.json"
+        },
+		{
+            "lang": "es",
+            "name": "Spanish",

```

Comment:
Shouldn't it say Español here instead of Spanish?

Reply:
Yes, I agree, but it isnt used, except in the Core translation, isnt it? I dont know if I must change it in all module translation I maintain (more than 20).

Reply:
I don't really know how that stuff is used honestly, I don't even know how to switch Foundry to another language if I wanted to test....
If you say it's not used anywhere and it's good, then I'll merge it as is, if you think it would be better to change it, then please do before I merge.
Changing it in all modules you maintain would be a chore, I don't think that would be needed, but if you decide to switch to Español, I'd suggest you just use that from now on, and change it in other modules only if you make other changes and if you remember about it... 
Hope that helps!

Reply:
Honestly, I dont know where does the names of the languages in the Core Configuration comes from, must be a core feature, because some are in the original language, others dont... I will ask, leave it in english and I will change if necessary in the next translation update

Reply:
Sounds good. Thanks for the feedback.. and for the translation of course :)